deploy/build-server/Dockerfile

```diff
@@ -0,0 +1,78 @@
+FROM ubuntu:18.04
+
+USER root
+SHELL ["/bin/bash", "-c"]
+# To make it easier for build and release pipelines to run apt-get,
+# configure apt to not require confirmation (assume the -y argument by default)
+ENV DEBIAN_FRONTEND=noninteractive
+RUN echo "APT::Get::Assume-Yes \"true\";" > /etc/apt/apt.conf.d/90assumeyes
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Comment:
It would be great to see the reverse-dependency a comment that required each of these to be brought in - otherwise, it would be very difficult to detect when we could remove a dependency.

E.g. 
\# Required by: Dockerfile
\# Required by: e2e tests